SDWebImage/SDImageCachesManager.m

```diff
@@ -17,6 +17,9 @@ @interface SDImageCachesManager ()
 @end
 
 @implementation SDImageCachesManager
+{
+    NSMutableArray<id<SDImageCache>> *caches_;
```

Comment:
Emm...A little idea about coding style. Can we change this ivar naming to a more property one, like `mutableCaches` ? Placing a suffix `_` looks a little strange for most of Objective-C developers. It's more like a C-style